src/ngraph/builder/norm.cpp

```diff
@@ -36,125 +37,147 @@ namespace ngraph
+        shared_ptr<Node> builder::opset1::l1_norm(const Output<Node>& value,
+                                                  const AxisSet& reduction_axes,
+                                                  float bias)
         {
-            shared_ptr<Node> values{
-                make_shared<op::Sum>(make_shared<op::Abs>(value), reduction_axes)};
+            const shared_ptr<Node> values{make_shared<ngraph::opset1::ReduceSum>(
+                make_shared<ngraph::opset1::Abs>(value),
+                ngraph::opset1::Constant::create(
+                    element::i64, Shape{reduction_axes.size()}, reduction_axes.to_vector()),
+                false)};
 
-            shared_ptr<Node> bias_node{
-                op::Constant::create(values->get_element_type(),
-                                     values->get_shape(),
-                                     vector<float>(shape_size(values->get_shape()), bias))};
+            const shared_ptr<Node> bias_node{ngraph::opset1::Constant::create(
+                values->get_element_type(),
+                values->get_shape(),
+                vector<float>(shape_size(values->get_shape()), bias))};
 
-            return (values + bias_node)->add_provenance_group_members_above({value});
+            return make_shared<ngraph::opset1::Add>(values, bias_node)
+                ->add_provenance_group_members_above({value});
         }
 
-        shared_ptr<Node> l2_norm(const Output<Node>& value,
-                                 const AxisSet& reduction_axes,
-                                 float bias,
-                                 BiasMode bias_mode,
-                                 bool keep_dims)
+        shared_ptr<Node> builder::opset1::l2_norm(const Output<Node>& value,
+                                                  const AxisSet& reduction_axes,
+                                                  float bias,
+                                                  BiasMode bias_mode,
+                                                  bool keep_dims)
         {
-            shared_ptr<Node> values{make_shared<op::v1::ReduceSum>(
-                value * value,
-                make_shared<op::Constant>(
+            shared_ptr<Node> values{make_shared<ngraph::opset1::ReduceSum>(
+                make_shared<ngraph::opset1::Multiply>(
+                    value, value, ngraph::op::AutoBroadcastType::NONE),
+                ngraph::opset1::Constant::create(
                     element::i64, Shape{reduction_axes.size()}, reduction_axes.to_vector()),
                 keep_dims)};
 
-            shared_ptr<Node> bias_node{
-                op::Constant::create(values->get_element_type(),
-                                     values->get_shape(),
-                                     vector<float>(shape_size(values->get_shape()), bias))};
+            shared_ptr<Node> bias_node{ngraph::opset1::Constant::create(
```

Comment:
We can relay on NumPy autobroadcasting mechanism.

Reply:
done